app/src/components/ArticleDetails/ArticleDetails.module.scss

```diff
@@ -0,0 +1,50 @@
+@use 'styles/spacings';
+
+.articleDetails {
+  padding: spacings.$spacing-m;
+  overflow: auto;
+  box-sizing: border-box;
+  overflow-y: auto;
+  font-size: 2.4rem;
+}
+
+.title {
+  font-size: 3.2rem;
```

Comment:
A good practice is to create a font file for the whole project in the "styles" folder. So are the colors and spacing.